core/services/ardupilot_manager/flight_controller_detector/linux/linux_boards.py

```diff
@@ -26,6 +31,15 @@ def check_for_i2c_device(self, bus_number: int, address: int) -> bool:
         except OSError:
             return False
 
+    def setup(self) -> None:
+        os.makedirs(LinuxFlightController.STANDARD_SCRIPT_DIRECTORY_PATH, exist_ok=True)
+        try:
+            os.symlink(
+                LinuxFlightController.STANDARD_SCRIPT_DIRECTORY_PATH, LinuxFlightController.LUA_SCRIPT_DIRECTORY_PATH
```

Comment:
```suggestion
                self.STANDARD_SCRIPT_DIRECTORY_PATH, self.LUA_SCRIPT_DIRECTORY_PATH
```


<img width="930" height="109" alt="Image" src="https://github.com/user-attachments/assets/bb970506-cb77-44b8-ad24-15235fde0c25" />